students/ahyounglee/users/models.py

```diff
@@ -0,0 +1,19 @@
+from django.db import models
+from django.core.validators import RegexValidator
+
+
+class User(models.Model):
+    username         = models.CharField(max_length=45, blank=False, null=False)
+    email            = models.EmailField(max_length=100, blank=False, null=False)
+    password         = models.CharField(max_length=256, blank=False, null=False)
+    phoneNumberRegex = RegexValidator(regex = r'^01([0|1|6|7|8|9]?)-?([0-9]{3,4})-?([0-9]{4})$')
```

Comment:
```suggestion
    password         = models.CharField(max_length=256, blank=False, null=False)
```

1. python에서 변수명은 카멜 케이스로 만들어야 합니다.
2. model의 class에서는 field 선언에 관련된 것만 넣어 주세요. validation과 관련된 로직은 view에서 처리하면 됩니다.
3. phone의 validators 속성도 제거해 주세요.

Reply:
수정했습니다.
확인 부탁드립니다.